getspice.py

```diff
@@ -198,6 +198,14 @@ def _download(self):
            HTTPS GET call to the NAIF server to download the required CSPICE
            distribution package.
         """
+        # Search proxy in ENV variables
+        proxy = False
+        try:
+            proxy = os.environ['HTTP_PROXY']
```

Comment:
so if a https_proxy is set and not an http one, this would just go the the except block, I think it makes more sense to try https first...

Reply:
You right. I changed it to store `*_proxy` env variables into `proxies` dict.